audio_load.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import pydub
+import librosa.core as lc
+from matplotlib import pyplot as plt
+
+def load_audio_from_files(file_list, file_format, length, fs):
```

Comment:
I'm not sure what `file_format` should be. Could you write a docstring like:
```
"""
Args:
  file_format: string, should be one of: wav, mp3
"""
```
please?